modules/trafficserver/templates/logrotate.erb

```diff
@@ -0,0 +1,8 @@
+<%= @paths['logdir'] %>/*.log {
```

Comment:
General note: ATS should log to syslog directly. In syslog-ng we can decide whether to forward to graylog, local log files, both, etc.

Reply:
ATS only supports sending system and emergencies to syslog.

See https://docs.trafficserver.apache.org/en/9.0.x/admin-guide/logging/destinations.en.html#syslog